index.html

```diff
@@ -85,242 +95,270 @@ <h2>LET'S CONNECT</h2>
+              />
+              </a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/hand-peace-icon.svg" alt="peace logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/twitter (2).svg" alt="twitter logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#footer-icon"
+                ><img src="./images/m-icon.svg" alt="m logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
+        <button  id="btn-get" class="see-pro .btn">Get my resume</button>
       </div>
-    </section>
-</fieldset>
+       <div class="desktopV">
+
+         <div class="frame-lang">
+           <h2 class="language">Languages</h2>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="list-skills-about">
+           <ul>
+             <li>
+              <img class="img-tanne" src="./images/javascript.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/JSdesktopAbout.svg" alt="" />
+             </li>
+             <li>
+              <img class="img-tanne" src="./images/html.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/HTMLdesktopAbout.svg" alt="" />
+            </li>
+             <li>
+              <img class="img-tanne" src="./images/css.svg" alt="" />
+            <img class="img-tanneDSKT" src="./images/CSSdesktopAbout.svg" alt="" />
+           </ul>
+         </div>
+         <div class="frame">
+           <p>Frameworks</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+         <div class="frame">
+           <p class="skill-desktop">Skills</p>
+           <img src="./images/Icon-Right-Larger.svg" alt="" />
+         </div>
+       </section>
+
+       </div>
+    <!-- ============CONTACT-ME-FORM==========  -->
+    <fieldset>
+      <section class="contact-me">
+       <div class="text-field hello">
+         <h2>Contact me</h2>
+         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+       </div>
+       <div class="input-field">
+         <form action="https://formspree.io/f/mnqynlww" method="post">
+           <input class="input-design" type="text" name="full_name" id="full_name" placeholder="Enter your Name" maxlength="30" required>
+           <input class="input-design" type="email" name="email" id="email" placeholder="Email Address" required>
+           <textarea class="textArea" name="message"  cols="30" rows="10" minlength="" maxlength="500" placeholder="write your message here" required></textarea>
+           <input  type="submit" value="Get in touch" class="see-pro submit-btn">
+         </form>
+         <div class="bottom_line"></div> 
+       </div>
+      </section>
```

Comment:
- [ ] I applaud you for implementing the contact form ☺️ . We are very close to making it professional. Here are some suggestions you may need to put into consideration to achieve the requirements. Kindly ensure that the section has a background as shown in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=2Q0MW8nBAbm41Ylg-0) by ensuring that you are using the right path directory.

   - Your design 😎 
![background](https://user-images.githubusercontent.com/99394374/215086391-92db48e5-a212-4814-b8a4-1b0b24e26cac.PNG)